ui/src/app/views/dataset-page.tsx

```diff
@@ -378,6 +380,33 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
       }
     }
 
+    isEllipsisActive(text) {
+      // For Data table with column title as active text, enable the tooltip if column's scroll
+      // width is greater than offsetWidth as that represents the textOverflow style has been
+      // applied and it has ellipsis at the end
+
+      const column = document.getElementsByClassName('p-column-title');
+      for (let i = 0; i < column.length; i++) {
```

Comment:
Rather than doing a loop here, it may make sense to generate a map at render time. Right now this is n^2, but generating a map at render time takes it to n and is probably simpler?